src/System.Management.Automation/help/HelpCommentsParser.cs

```diff
@@ -354,7 +354,7 @@ internal XmlDocument BuildXmlFromComments()
                     // The title is automatically generated
                     XmlElement title = _doc.CreateElement("maml:title", mamlURI);
                     string titleStr = string.Format(CultureInfo.InvariantCulture,
-                        "				-------------------------- {0} {1} --------------------------",
+                        "\t\t\t\t-------------------------- {0} {1} --------------------------",
```

Comment:
Is the use of tab a problem #6313?

Reply:
Don't know.
For this specific change, it keeps the original behavior because the removed whitespaces are essentially 4 tabs.

Reply:
@SteveL-MSFT Could you please comment - is this related to #6313?

Reply:
@iSazonov This is not a blocking comment, right? This change itself doesn't alter the original behavior. Even if the use of tab is proven to be the cause of #6313, I think it should be dealt with in a separate PR.

Reply:
Tabs in general are problematic as their size depends on the console.  Try this on macOS or Linux:

```powershell
tabs 6,10,12
"a`tb`tc`td"
```

If the intent of tabs in strings is to indent, I think it's better to just use four spaces.  This PR is unrelated to #6313 expect for both related to behavior of tabs at the console.

Reply:
I opened #6381 to track the review of the use of '\t' here.